lib/lago_http_client/lago_http_client/client.rb

```diff
@@ -114,7 +114,7 @@ def get(headers: {}, params: nil, body: nil)
     attr_reader :http_client
 
     def raise_error(response)
-      raise(::LagoHttpClient::HttpError.new(response.code, response.body, uri))
+      raise(::LagoHttpClient::HttpError.new(response.code, response.body, uri, response_headers: response.to_hash))
```

Comment:
can't we just fetch headers? I guess we are sending whole object as headers...

Reply:
I did an improvement, but I'm not sure what exactly you mean by fetching just headers?